### ISSUE TITLE
Requires Tatin to intialise before opening project

### ISSUE THREAD

**Issue by rikedyp** [aplteam/Cider#18]
Don't know whether this is documented and I missed it, but

```
      ]CIDER.OpenProject \g\ParquetDotNet
* Command Execution Failed: VALUE ERROR: Undefined name: Tatin
```

goes away if I first do 

```
      ]TATIN.Version
┌─────┬───────────┬──────────┐
│Tatin│0.90.0+1485│2023-02-27│
└─────┴───────────┴──────────┘
```

I would expect not to have to call a different UCMD before being able to use Cider

**Comment by aplteam** [comment]
Tatin it's will be part of a 19.0 standard installation. In older versions it's up to the user to make sure that it's loaded into `⎕SE`.

However, the error message should be more explicit about what's required.

**Comment by aplteam** [comment]
Changed my mind. Solved in version 0.23.4
